Tests/InaraTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using EddiInaraService;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using UnitTests;
+
+namespace IntegrationTests
+{
+    [TestClass]
+    public class InaraTests : TestBase
+    {
+        [TestInitialize]
+        public void start()
+        {
+            MakeSafe();
+        }
+
+        [TestMethod]
+        public void TestSendEventBatch()
+        {
+            List<InaraAPIEvent> inaraAPIEvents = new List<InaraAPIEvent>()
+            {
+                { new InaraAPIEvent(DateTime.UtcNow, "getCommanderProfile", new Dictionary<string, object>() { { "searchName", "No such name" } })},
+                { new InaraAPIEvent(DateTime.UtcNow, "getCommanderProfile", new Dictionary<string, object>() { { "searchName", "Artie" } })}
+            };
+            List<InaraResponse> responses = InaraService.Instance.SendEventBatch(ref inaraAPIEvents, true);
```

Comment:
Please remove the 2nd argument here to correct the current failing build status in Appveyor. 
Note that the `SendEventBatch` method will still always fail with the current block in `SendEventBatch()`, as `MakeSafe()` sets EDDI as if the game were in beta.

Reply:
Resolved I believe.